python/cucim/src/cucim/skimage/transform/_geometric.py

```diff
@@ -1510,7 +1517,7 @@ class SimilarityTransform(EuclideanTransform):
 
     where ``s`` is a scale factor and the homogeneous transformation matrix is::
 
-        [[a0  b0  a1]
+        [[a0  -b0  a1]
```

Comment:
Was this a typo or was there some kind of behavior change?

Reply:
it was just a typo fix